src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -45,6 +46,9 @@ public void onStart(ITestContext context) {
                         .put("Browser.Version", "96.0.4664.93 (Official build), (64 bit)")
```

Comment:
in case of different browser these env details in allure will be invalid

Reply:
+